Frameworks/Foundation/NSURLResponse.mm

```diff
@@ -134,29 +134,75 @@ - (id)copyWithZone:(NSZone*)zone {
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
 + (BOOL)supportsSecureCoding {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return YES;
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
-- (id)initWithCoder:(NSCoder*)decoder {
-    UNIMPLEMENTED();
-    return StubReturn();
+- (instancetype)initWithCoder:(NSCoder*)coder {
+    if (self = [super init]) {
+        _expectedContentLength = [coder decodeIntForKey:@"NS.expectedContentLength"];
```

Comment:
>NS.expectedContentLength [](start = 58, length = 24)

Nit:Could we define constants for these and use them in encode/decode functions?